src/orders/README.md

```diff
@@ -206,6 +206,39 @@ This creates two secrets managed by kubernetes, named `atp-db-creds` and `atp-jk
 The kubernetes deployment manifest `atp-spring-jpa.yml` references these secrets and provides them to the runtime environment of the containers. The application now references the properties through the environment variables instead of reading them from a file. Alternatively the config files themselves could have been provided and mounted as volumes in to the container.
 
 
+```bash
+kubectl create secret generic monitoring-secret \
+--from-literal=compartment_id="<your compartment_id>" \
+--from-literal=tenant_id="<your tenancy_id>" \
+--from-literal=user_id="<your user_id>" \
+--from-literal=fingerprint="<your fingerprint>" \
+--from-literal=passphrase="<your passphrase>" \
+--from-literal=monitoring_endpoint="<your monitoring endpoint>" \
+--from-literal=region="<your region>" \
+--from-file=oci_api_key=<your oci_api_key.pem>
+```
+
+The above secret is required to send out custom metric (payment failure) to OCI Monitoring using OCI Java SDK.
+
+### For Developer setup:
+
+# Build
+
+## Java: building application jar
+
+OCI-java-sdk is a dependency and needs to be manually uploaded to 
```

Comment:
Like before - this adds significant friction to the build process and all users are now forced to do this. We should likely split off the monitoring in to a separate service that's optional

Reply:
that sounds like a better option.

Reply:
this is only for dev env on laptop. For user's it is pre-built (DockerFile)